src/eterna/pose2D/PoseField.ts

```diff
@@ -15,9 +16,10 @@ export default class PoseField extends ContainerObject implements KeyboardListen
 
     private static readonly SCORES_POSITION_Y = 128;
 
-    constructor(edit: boolean) {
+    constructor(edit: boolean, annotationManager: AnnotationManager) {
         super();
-        this._pose = new Pose2D(this, edit);
+        this._pose = new Pose2D(this, edit, annotationManager);
+        this._annotationManager = annotationManager;
```

Comment:
For future reference, would you mind sharing why we prefer to move the `annotationManager` state to `PoseField`?

Reply:
The main intent here is to be able to pass it to Pose2D as a constructor argument so that it doesn't need to be nullable (and to guarantee we'll have it as soon as we need it). I kept a reference to it on poseField since we need it there and are given a reference to it anyways (and to reduce the public interface of Pose2D)

Reply:
Got it!